pkgs/development/python-modules/sphinx-tippy/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  flit-core,
+  beautifulsoup4,
+  jinja2,
+  requests,
+  sphinx,
+  sphinxHook,
+  furo,
+  myst-parser,
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-tippy";
+  version = "0.4.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "sphinx_tippy";
+    inherit version;
+    hash = "sha256-JVq+4K7YCF/bmrDMWVzXpFdIrn9GYhVuUAoXo/c61j0=";
+  };
+
+  build-system = [
+    flit-core
+  ];
+
+  nativeBuildInputs = [
+    sphinxHook
+    furo
+    myst-parser
+  ];
+
+  dependencies = [
+    beautifulsoup4
+    jinja2
+    requests
+    sphinx
+  ];
+
```

Comment:
add pytestCheckHook please

Reply:
There's only one test, that is currently broken, unfortunately…